src/nsresourced/userns-restrict.c

```diff
@@ -218,7 +218,7 @@ int userns_restrict_put_by_inode(
 
                 r = sym_bpf_map_update_elem(outer_map_fd, &ino, &inner_map_fd, BPF_ANY);
                 if (r < 0)
-                        return log_debug_errno(errno, "Failed to replace map in inode hash: %m");
+                        return log_debug_errno(r, "Failed to replace map in inode hash: %m");
```

Comment:
These aren't strictly "mismatched", since libbpf both returns errno value and sets `errno` var. So either way would work.

Reply:
Well, the description in `libbpf.h` doesn't say anything about `errno`. But if you want to leave this as it is, I'm fine with that.

Reply:
I like the current form better :p